apps/cyberstorm-storybook/LinkLibrary.tsx

```diff
@@ -0,0 +1,31 @@
+import { LinkLibrary, ThunderstoreLinkProps } from "@thunderstore/components";
+
+interface LinkProps extends ThunderstoreLinkProps {
+  url: string;
+}
+
+const Link = (props: LinkProps): React.ReactElement => {
+  const { url, children } = props;
+
+  // Render links as spans so clicking them does nothing.
```

Comment:
Doesn't really look like this renders a span 😄 

Anyways, I'm not sure there's a purpose to have this component anymore. The LinkLibrary definition below could return `<a>` elements directly. But feel free to keep it, if you feel it improves legibility oslt.

Reply:
Yeah, seems like it's not used so I removed it.

Reply:
Has something gone wrong with git here? At least "Files Changed" in GitHub still shows the outdated code comment here.

But, if my suggestion in earlier comment about ditching the `LinkinProvider` comes to pass, this file can be removed too.

Reply:
Removed the LinkingLibrary